sonar-db/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -527,21 +526,23 @@ public void select_projects_from_view() {
   public void select_projects() {
     OrganizationDto organization = db.organizations().insert();
     ComponentDto provisionedProject = db.components().insertProject();
-    ComponentDto provisionedView = db.components().insertView(organization, (dto) -> {});
+    ComponentDto provisionedView = db.components().insertView(organization, (dto) -> {
+    });
     String projectUuid = db.components().insertProjectAndSnapshot(ComponentTesting.newProjectDto(organization)).getComponentUuid();
     String disabledProjectUuid = db.components().insertProjectAndSnapshot(ComponentTesting.newProjectDto(organization).setEnabled(false)).getComponentUuid();
     String viewUuid = db.components().insertProjectAndSnapshot(ComponentTesting.newView(organization)).getComponentUuid();
 
     assertThat(underTest.selectProjects(dbSession))
-        .extracting(ComponentDto::uuid)
-        .containsOnly(provisionedProject.uuid(), projectUuid);
+      .extracting(ComponentDto::uuid)
+      .containsOnly(provisionedProject.uuid(), projectUuid);
   }
 
   @Test
   public void select_provisioned() {
     OrganizationDto organization = db.organizations().insert();
     ComponentDto provisionedProject = db.components().insertComponent(newProjectDto(organization).setKey("provisioned.project").setName("Provisioned Project"));
-    ComponentDto provisionedView = db.components().insertView(organization, (dto) -> {});
+    ComponentDto provisionedView = db.components().insertView(organization, (dto) -> {
```

Comment:
FYI, an override of `insertView` without a second argument exists
(it's legacy code, I know)